raiden/api/python.py

```diff
@@ -387,10 +404,19 @@ def channel_open(
                 token_network.new_netting_channel(
                     partner=partner_address,
                     settle_timeout=settle_timeout,
-                    given_block_identifier=views.state_from_raiden(self.raiden).block_hash,
+                    given_block_identifier=given_block_identifier,
                 )
             except DuplicatedChannelError:
                 log.info("partner opened channel first")
+            except RaidenRecoverableError:
```

Comment:
Is the above `except DuplicatedChannelError` still valid?

Reply:
Yes, the problem with `TokenNetwork` raising sometimes either type of exception remains...